zigbee/Tutorials-and-Examples/Zigbee-Steup-Zigbee-Network-Without-Association.md

```diff
@@ -0,0 +1,237 @@
+
+
+# 1. Introduction
+This article will provide a guideline for how to setup Zigbee network without association process.
+
+# 2. Joining a Network Through Association
+As you probably know that setup a Zigbee network always required to establishing a new network on the Zigbee Coordinator side and permit device to join, and then the joining device should initiate the procedure for network discovery to find a joinable network. The beacon request will be sent out from the joining device, and router or coordinator will response the beacon request with beacon. 
+
+Each beacon frame includes information such as the PAN ID, extend PAN ID, the information that the device is capable of accepting join request from new device. 
+
+The joiner device receives beacons from nearby routers, and it will choose a network with the best signal quality to join from the discovered networks. Once the network is selected, it will send Association request. In this request, the PAN ID is set to the chosen PAN and the destination node id is set to the node ID of the chosen device, and the capability information of the joining device will also be carried on in the Association request command frame.
```

Comment:
"Chosen device" can we specify what it refer to?

Reply:
chosen device -> chosen device which sent the beacon